static/stylesheet/style.less

```diff
@@ -430,7 +430,7 @@ ul.social {
     background-color: @github-bg-color;
   }
 
-  a.sc-envelope-o {
+  a.sc-envelope-open {
```

Comment:
not sure if this is going to break something

Reply:
In the new `font-awesome` there is no `envelope-o`, but `envelope-open`, so the migration would require changing the name. I can't find any place which would be broken by this.

Reply:
it's not in Flex source, it's in pelicancong.py, when you choose to show your email as an icon.